btrack/napari/widgets/_hypothesis.py

```diff
@@ -22,7 +20,7 @@ def _create_hypotheses_widgets() -> dict[str, tuple[str, QtWidgets.QWidget]]:
     ]
 
     widget = QtWidgets.QListWidget()
-    widget.addItems(hypotheses)
+    widget.addItems([f"{h.replace('_', '(')})" for h in hypotheses])
```

Comment:
I would have liked to make it bold here `<b>P</b>()` kind of thing but it seems non-straightforward for `QListWidgetItem`

Reply:
So I think it is possible, but you have to do something like this https://stackoverflow.com/a/53572278/7359333 which seems very involved